appveyor.yml

```diff
@@ -10,13 +10,15 @@ environment:
   matrix:
     - RuntimeURL: https://atom.io/download/atom-shell
       RuntimeName: iojs
-      RuntimeVersion: v2.0.4
+      RuntimeVersion: v2.0.6
+      ElectronVersion: "2.0.6"
```

Comment:
This should probably be removed and RuntimeVersion just be placed in quotations where necessary. Environment variables with quotation marks are an occupational hazard as well.

Reply:
This is needed for the npm install -g electron@$Env:ElectronVersion command. There is no "v2.0.6" of electron, only "2.0.6".